internal/rest/resources/control.go

```diff
@@ -129,12 +129,12 @@ func joinWithToken(state state.State, r *http.Request, req *internalTypes.Contro
 
 		cert, err := shared.GetRemoteCertificate(url.String(), "")
 		if err != nil {
-			return response.SmartError(fmt.Errorf("Failed to get certificate of cluster member %q: %w", url.URL.Host, err))
+			logger.Debug("Failed to get certificate of cluster member", logger.Ctx{"address": url.String(), "error": err})
```

Comment:
Should this be logger.Warn or logger.Info? Debug seems too low.

Reply:
True, increased to `Warn`